apps/idam/idam-web-admin/ithc.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: idam-web-admin
+  namespace: idam
+spec:
+  releaseName: idam-web-admin
+  values:
+    java:
+      image: hmctspublic.azurecr.io/idam/web-admin:preview-7f70580-20220314172543
```

Comment:
Please create and assign imagepolicy to this `image`

Reply:
We don't deploy using image policy to environments other than preview. We have more of a scheduled image release than an automated image release due to how our release schedule works as a dependency for most other teams.